src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -1372,6 +1394,15 @@ export class NetworkAreaDiagramViewer {
 
     public checkAndUpdateLevelOfDetail(svg: SVGSVGElement) {
         const maxDisplayedSize = this.getCurrentlyMaxDisplayedSize();
+        const previousMaxDisplayedSize = this.getPreviousMaxDisplayedSize();
+        // in case of bad or unset values NaN or Infinity, this condition is skipped and the function behaves as if zoom changed
+        if (
+            Math.abs(previousMaxDisplayedSize - maxDisplayedSize) / previousMaxDisplayedSize <
```

Comment:
divide by zero

Reply:
in case of bad or unset values NaN or Infinity, this condition is skipped and the function behaves as if zoom changed